.palantir/autorelease.yml

```diff
@@ -0,0 +1,4 @@
+version: 3
+schedules:
+  timer:
+    time_since_last_release: 14 days
```

Comment:
I do not think we should eggtime a library like this every 14 days. Instead we should find some other trigger to autorelease only when necessary. For example, if the palantir-java-format is upgraded in the repo. This may require a custom excavator to do.

Reply:
Aren't the autorelease policies set at the organization level? I'd prefer not to deviate from the default.